test/libsolidity/ASTJSON/documentation_local_variable.sol

```diff
@@ -0,0 +1,25 @@
+contract C {
+	function f() public pure {
+		/// A documentation tag for x
+		uint x = 1;
+		for (
+			/// documentation for i
+			uint i = 0;
+			i < 10;
+			++i
+		) {
+			/// documentation for j
+			uint j = 0;
+		}
+	}
+	function g(
+		/// documentation for param1
+		uint param1,
+		/// documentation for param2
+		uint param2,
+		/// documentation for param3
+		uint param3
+	) public {}
+}
```

Comment:
In this test, only the documentation for `uint x = 1` and `uint j = 0` shows up in the json file.

Reply:
Can you check why?

Reply:
We need to export `StructuredDocumentation` for this to show up. See https://gitter.im/ethereum/solidity-dev?at=6065e3aad765936399ca3939.